File: topic_4/Python/2-max3.py
# Recebe três números e retorna o maior valor entre eles

def max3(a, b, c):
    if(a > b):
        if(a > c):
            return a
        else:
            return c
    else:
        if(b > c):
            return b
        else:
            return c

print(max3 (6, 2, 4) == 6) #True
print(max3 (6, 7, 4) == 7) #True
print(max3 (6, 7, 9) == 9) #True
print(max3 (5, 2, 5) == 5) #True

File: topic_4/Haskell/2-max3.hs
-- Recebe três números e retorna o maior valor entre eles
max3 a b c
    | a > b && a > c = a
    | a > b && a < c = c
    | b > c && a < b = b
    | otherwise = c

main = do
    a <- readLn :: IO Int
    b <- readLn :: IO Int
    c <- readLn :: IO Int
    print $ max3 a b c